I have submitted my week 5 site deweycheetemandhowe.html

JennHopkins opened the pull request:
Here is the best I can do for the things requested of me.

Also, updated EZbreezy (and adoptadog).

See, I'm good for something.

JennHopkins committed:
added calulator.js file and made changes to it

JennHopkins committed:
updated calculator.js

JennHopkins committed:
added countdown.js

JennHopkins committed:
Add files via upload

JennHopkins committed:
Add files via upload

JennHopkins committed:
Add files via upload

JennHopkins committed:
added file

JennHopkins committed:
Merge branch 'master' of https://github.com/JennHopkins/html200-winter2020

JennHopkins committed:
modified code to correct errors

JennHopkins committed:
modified and corrected exercise two

JennHopkins committed:
added ezBreezy.js and index.html for lesson 4

JennHopkins committed:
updated exercise-three.js

JennHopkins committed:
updated dch.html and JS file

jimmynono commented on deweycheetemandhowe.html:
```diff
@@ -0,0 +1,57 @@
+      </ul>
+      <button onclick = "bankAction()">Sucker!</button>
+      <p>Hello, Bankee, what would you like D,C and H Money Banks do to you today?</p>
+    <script>
```
You'll always want to do all your JS work in JS files, not inline in your HTML

jimmynono commented on deweycheetemandhowe.html:
```diff
@@ -0,0 +1,57 @@
+    alert('Your new balance is now' + '$' + result);
+   }
+
+   function depositing() {
```
Each time you call this function it resets the balance to 500 (the withdrawl too)

JennHopkins committed:
added js file and updated html file

JennHopkins commented:
Okay, I have re-done the bank app - still not happy with the "balance" portion of it (gonna ask my friend about that), but everything works exactly as it's supposed to.

Here you go!

JennHopkins commented:
By the way - I was surprised to learn about the "while loop".  Nobody told me that a "while loop" could have a comparison condition. EVERY single example I found online and even in this class made it look like it could only do a numerical value for its condition statement. I thought that it's proper syntax was: while(variable < 3) { //do this code}; Because every single example had the greater than (or less than) symbol and then a number value. I did experiment with an equal sign operator: while(variable = d) {//do this code}; but it didn't make anything happen, I therefore assumed (incorrectly) that it wouldn't work without the greater than (or less than) symbol and a number - but I had no other source of information to tell me otherwise. Well, now I know.

JennHopkins committed:
made updates to dch.js

JennHopkins commented:
Okay, so I have fixed the balance function so that it will show you how much you have either withdrawn or deposited. I have also added a transfer function (though you have to blame my friend Roxanne for that, she talked me into it). Oh Roxy, look what you made me do!
